psi4/src/psi4/libfock/cubature.h

```diff
@@ -156,7 +156,7 @@ class MolecularGrid {
     /// Maximum number of funtions in a block
     int max_functions() const { return max_functions_; }
     /// Total collocation size of all blocks
-    size_t collocation_size() { return collocation_size_; }
+    const size_t collocation_size() { return collocation_size_; }
```

Comment:
sorry, I mean the function should be const (doesn't change any members), not the return value (which is returned by value anyway, so the const doesn't apply)

Reply:
Oh, sure I thought that was a bit weird. We need to const a very large number of functions in Psi4. Is it worth doing it for this one off case without changing all of cubature or libfock?

Reply:
Yeah, there are many const qualifiers missing for sure.  Was just pointing it out so that we can try to add correct code going forwards, but it's not a big deal if we skip this one.  Just reviewing what I see, that's all